### PR TITLE
added resource in readme by numfocus

### LINKED ISSUE #525: Add "Getting Started with Open Source" by NumFocus
https://github.com/numfocus/getting-started-with-open-source

### PR DESCRIPTION
**Pull request by vj-codes**
Fixes #525

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ If you have questions or comments, please create an issue.
 - ["How to Contribute to Open Source Project" by Maryna Z](https://rubygarage.org/blog/how-contribute-to-open-source-projects) - This comprehensive article is directed towards businesses (but still useful for individual contributors) where it talks about why, how, and what open-source projects to contribute to.
 - ["start-here-guidelines" by Andrei](https://github.com/zero-to-mastery/start-here-guidelines) -
 Lets Git started in the world of opensource, starting in the opensource playground. Especially designed for education and practical experience purposes.
+- ["getting-started-with-open-soruce" by NumFocus](https://github.com/numfocus/getting-started-with-open-source) - a Github repo that helps contributors overcoming barriers in open-source.
```
Comment (erictleung) [inline README.md:69]: Great work! Just a slight suggestion to make the format consistent with the rest of the section titles and there was a slight typo too.

```suggestion
- ["Getting Started with Open Source" by NumFocus](https://github.com/numfocus/getting-started-with-open-source) - a GitHub repo that helps contributors overcome barriers to entry in open-source.
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Thanks for helping! Just a small comment on changes.

**Comment by vj-codes** [comment]
@ericboucher done!

**Review by erictleung — APPROVED** [review]
LGTM! 👍 Thanks for taking the time to add this resource! :sparkles:

**Pull request merged**